.gitignore

```diff
@@ -7,4 +7,4 @@
 /pkg/
 /spec/reports/
 /tmp/
-*.gem
```

Comment:
I don't think we'd want to be checking in `.gem` files.

Reply:
Whups, fixed!